utils/installer/install.sh

```diff
@@ -85,6 +85,25 @@ function msg() {
   printf "%s\n" "$text"
 }
 
+function confirm() {
+  local question="$1"
+  while true ; do
+    msg "$question"
+    read -p "[y]es or [n]o (default: no) : " -r answer
+    case "$answer" in
+      y | Y | yes | YES | Yes)
+        return 0
+        ;;
+      n | N | no | NO | No)
```

Comment:
```suggestion
      n | N | no | NO | No | *[[:blank:]]* | "")
```
this seems to work for me, wdyt? but we still need to test it on macOS, just in case, although I believe this is POSIX compliant.

Reply:
Works fine!